README.md

```diff
@@ -25,13 +25,46 @@ For building the Nighthawk test server, see [here](source/server/README.md).
 ### Test it
 
 ```bash
-bazel test -c dbg //test:nighthawk_test
+bazel test -c dbg //...
 ```
 
 ### Build it
 
 ```bash
-bazel build -c opt //:nighthawk_client
+bazel build -c opt //...
+```
+### Using the service
```

Comment:
I would move this to after the CLI section and call it `### Nighthawk gRPC service`